modules/rostests/apitests/shell32/closewnd.h

```diff
@@ -17,4 +17,5 @@ void GetWindowList(PWINDOW_LIST pList);
 void GetWindowListForClose(PWINDOW_LIST pList);
 HWND FindNewWindow(PWINDOW_LIST List1, PWINDOW_LIST List2);
 void CloseNewWindows(PWINDOW_LIST List1, PWINDOW_LIST List2);
```

Comment:
Side-note: these "List1", "List2" names could be improved in order for the user to understand what's the purpose of "List1", with respect to "List2", ...